package.json

```diff
@@ -10,7 +10,7 @@
     "lint:fix": "eslint . --ext .ts --fix",
     "format": "prettier --check './src/**/*.ts'",
     "format:fix": "prettier --write './src/**/*.ts'",
-    "test": "jest"
+    "test": "jest --silent"
```

Comment:
The `--silent` flag will stop messages from kintone-cli from being printed on the console, keeping the console clean.